sdk/automation/Azure.ResourceManager.Automation/tests/SessionRecords/WebhookTests/CreateOrUpdateExistGetGetAllAsync.json

```diff
@@ -0,0 +1,529 @@
+      },
+      "ResponseBody": {
+        "id": "/subscriptions/db1ab6f0-4769-4b27-930e-01e2ef9c123c/resourceGroups/testRG-3051/providers/Microsoft.Automation/automationAccounts/automation1670/webhooks/webhook8001",
+        "name": "webhook8001",
+        "type": "Microsoft.Automation/AutomationAccounts/Webhooks",
+        "properties": {
+          "isEnabled": false,
+          "expiryTime": "2023-05-25T06:08:34.0928233\u002B00:00",
+          "runbook": {
+            "name": "runbook7879"
+          },
+          "lastInvokedTime": null,
+          "runOn": null,
+          "parameters": null,
+          "uri": "",
+          "creationTime": "2023-05-24T06:08:37.4961829\u002B00:00",
+          "lastModifiedBy": "",
+          "lastModifiedTime": "2023-05-24T06:08:37.4961829\u002B00:00"
+        }
+      }
+    },
+    {
+      "RequestUri": "https://management.azure.com/subscriptions/db1ab6f0-4769-4b27-930e-01e2ef9c123c/resourceGroups/testRG-3051/providers/Microsoft.Automation/automationAccounts/automation1670/webhooks/webhook8001?api-version=2015-10-31",
+      "RequestMethod": "GET",
+      "RequestHeaders": {
+        "Accept": "application/json",
+        "Authorization": "Sanitized",
+        "traceparent": "00-3aed3e996fae188fecb2f4f6a470e127-b8d69366c977d626-00",
+        "User-Agent": "azsdk-net-ResourceManager.Automation/1.1.0-alpha.20230524.1 (.NET 7.0.5; Microsoft Windows 10.0.22621)",
+        "x-ms-client-request-id": "f4c78a1f4bdd44d9bebcbf16ba1b612e",
+        "x-ms-return-client-request-id": "true"
+      },
+      "RequestBody": null,
+      "StatusCode": 200,
+      "ResponseHeaders": {
+        "Cache-Control": "no-cache",
+        "Content-Length": "556",
+        "Content-Type": "application/json; charset=utf-8",
+        "Date": "Wed, 24 May 2023 06:08:40 GMT",
+        "Expires": "-1",
+        "Pragma": "no-cache",
+        "Server": "Microsoft-HTTPAPI/2.0",
+        "Strict-Transport-Security": "max-age=31536000; includeSubDomains",
+        "X-Content-Type-Options": "nosniff",
+        "x-ms-correlation-request-id": "43b65a6f-09d1-4427-99f7-bcbf84706bae",
+        "x-ms-ratelimit-remaining-subscription-reads": "11996",
+        "x-ms-request-id": "66965836-4ad2-4cfe-912e-e431a7026aae",
+        "x-ms-routing-request-id": "SOUTHEASTASIA:20230524T060841Z:43b65a6f-09d1-4427-99f7-bcbf84706bae"
+      },
+      "ResponseBody": {
+        "id": "/subscriptions/db1ab6f0-4769-4b27-930e-01e2ef9c123c/resourceGroups/testRG-3051/providers/Microsoft.Automation/automationAccounts/automation1670/webhooks/webhook8001",
+        "name": "webhook8001",
+        "type": "Microsoft.Automation/AutomationAccounts/Webhooks",
+        "properties": {
+          "isEnabled": false,
+          "expiryTime": "2023-05-25T06:08:34.0928233\u002B00:00",
+          "runbook": {
+            "name": "runbook7879"
+          },
+          "lastInvokedTime": null,
+          "runOn": null,
+          "parameters": null,
+          "uri": "",
```

Comment:
@dvbb Out of curiosity, did you manually add this to reproduce the issue?

Reply:
Yes, Service will return `"uri":""` or `"uri":null` in all GET, LIST methods, and SDK side will throw uri format issue when  Service  return `"uri":""`